documentation/documentation/docs/intro.md

```diff
@@ -0,0 +1,47 @@
+---
+sidebar_position: 1
```

Comment:
this is a guide for docusaurus, maybe don't keep this on the repo

Reply:
I'll keep the page for now with something basic about our website, it can be helpfull to have later

Reply:
https://github.com/SELab-2/UGent-3/pull/250/commits/c2af2565e6508f3e8caa588f821307480a79c078